backend/git_real/models.py

```diff
@@ -123,8 +126,34 @@ def create_or_update_from_github_api_data(cls, repo, pull_request_data):
         pull_request, _ = cls.get_or_create_or_update(
             repository=repo, number=number, defaults=defaults, overrides=defaults
         )
+
+        if pull_request_data['base']['repo']['pushed_at'] == pull_request_data['updated_at']:
```

Comment:
This comparison also seems odd. Is this an accurate way to check if it's a push event? 

What if a person pushes code, scratches their nose, and then only opens a PR? 
Is `pushed_at` equal to the last push of any branch, or the last push on the main branch? 

It seems like there are a lot of assumptions happening here. It would be better to just look at the json passed to the webhook.